app/views/bulk_url_upload_results_mailer/results_email.html.erb

```diff
@@ -13,7 +13,7 @@
   </p>
 <% else %>
   <p>
-    <%= @results.ok_count %> URLs were created successfully.
+    <%= @results.ok_count %> URLs were created or enqueued successfully.
```

Comment:
If necessary, a future story can list the separate numbers for created vs. enqueued, but that is out of scope for these stories.